test/cli.js

```diff
@@ -10,3 +12,23 @@ test('runs benchmark', async (t) => {
 
   t.regex(result.stdout, /[0-9,]+ ops\/sec ±[0-9.]+% \(\d+ runs sampled\)/)
 })
+
+test('throttles CPU', async (t) => {
+  const getOpsSec = (resultString) => {
+    return parseInt(
+      resultString.match(/([\d,]+) ops\/sec/)[1].replace(/,/g, '')
+    )
+  }
+  const binPath = path.resolve(__dirname, '../lib/cli.js')
+  const fixturePath = path.resolve(__dirname, 'fixtures/benchmark.js')
+
+  const woutT = (await execa(binPath, [fixturePath])).stdout
+  await sleep(100) // sometimes port is still in use
```

Comment:
Is it possible to wait for the actual port to be available? Otherwise I can see this being a potentially flaky test.

Reply:
Nice catch!

I also do not like this sleep, because it is really a potential for a flaky test, so let me think about it for a moment.

A bigger question why port is not opened after Promise is resolved

Reply:
@Rowno Pushed a fix

Instead of developing some watcher to wait for the port to be opened, I've found a reason why cli was not waiting for the puppeteer to completely close Chrome.

Fixed this problem and ran the tests 12 times in a row to be sure it never fails 💪

Reply:
<img width="752" alt="Screen Shot 2022-07-19 at 19 04 21" src="https://user-images.githubusercontent.com/3503252/179797081-0f5e08b2-5708-43a9-8c8b-a40dda0f5d8a.png">

Reply:
Sweet